daskml/xgboost.py

```diff
@@ -0,0 +1 @@
+from dask_xgboost import *  # noqa
```

Comment:
Thoughts on making xgboost optional? AFAICT, xgboost doesn't upload wheels.

Reply:
I think that it is.  We don't import this in `daskml/__init__`